api/src/shared/storage/infrastructure/providers/S3ObjectStorageProvider.js

```diff
@@ -71,6 +71,11 @@ class S3ObjectStorageProvider {
     const getObjectCommand = new this.#dependencies.clientS3.GetObjectCommand({ Bucket: this.#bucket, Key: key });
     return this.#dependencies.s3RequestPresigner.getSignedUrl(this.#s3Client, getObjectCommand, { expiresIn });
   }
+
+  async downloadFile({ key }) {
```

Comment:
Etre plus explicite:
```suggestion
  async readFile({ key }) {
```

Reply:
```suggestion
  readFile({ key }) {
```

Reply:
Done